src/test/java/com/cloudbees/jenkins/support/filter/ContentMappingsTest.java

```diff
@@ -108,6 +113,8 @@ public void contentMappingsSurviveSerializationRoundTrip() {
     @LocalData
     public void jenkinsVersionIncludedAsStopWord() {
         rr.then(r -> {
+            String version = Jenkins.VERSION;
```

Comment:
Shouldn't this be done by the before and after methods?

Reply:
This could be done in a before method, but since there's only one place that cares I decided it wasn't particularly worthwhile to do so. It would mean calling reset way more often than anything cared.

Reply:
But there are already before and after methods doing so, so either those or this is redundant?